### PR TITLE
Update restore

### PR DESCRIPTION
**Pull request by ericgaspar**
-Fix linter warning

### CODE REVIEW

**Comment by ericgaspar** [comment]
!testme

File: conf/arm64.src [hunk 1]
```diff
@@ -3,4 +3,4 @@ SOURCE_SUM=5e98df2198e7dfe6479d565e82fd3e68a07d2334435a05c327e69de4264d54f7
 SOURCE_SUM_PRG=sha256sum
 SOURCE_FORMAT=zip
 SOURCE_IN_SUBDIR=false
-#SOURCE_FILENAME=
+SOURCE_FILENAME=mautrix-whatsapp
```
Comment (Gredin67) [inline conf/arm64.src:6]: As is `amd64.src`
should we add `SOURCE_EXTRACT=false`
and remove `SOURCE_FORMAT=zip`
or is it explicit?

File: conf/armhf.src [hunk 2]
```diff
@@ -3,4 +3,4 @@ SOURCE_SUM=8183e7236d6568f35c2276a74187e75d706c6479ef08a49b929d4bfa0c59c0b5
 SOURCE_SUM_PRG=sha256sum
 SOURCE_FORMAT=zip
 SOURCE_IN_SUBDIR=false
```
Comment (Gredin67) [inline conf/armhf.src:5]: same here

**Review by Gredin67 — APPROVED** [review]
lgtm apart from these modifications

**Comment by Gredin67** [comment]
Thinking about simplifying maintenance of the mautrix briges, should we check that this clean up and the one you did for mautrix-signal https://github.com/YunoHost-Apps/mautrix_signal_ynh/pull/27 are homogeneous ?

Then we could propagate signal to the other mautrix python bridges easily, as proposed here: https://github.com/YunoHost-Apps/mautrix_facebook_ynh/issues/4

**Comment by Gredin67** [comment]
!testme

**Pull request merged**